helm-charts/codegen/templates/ui-nginx.yaml

```diff
@@ -22,13 +22,19 @@ metadata:
   labels:
     {{- include "codegen.labels" . | nindent 4 }}
 data:
+  {{- if (index .Values "data-prep" "enabled") }}
```

Comment:
```suggestion
  {{- if .Values. data-prep.enabled }}
```

Reply:
@lianhao I'm seeing the following error which is resolved by using `index`:

`
Error: parse error at (codegen/templates/ui-nginx.yaml:25): bad character U+002D '-'
│
│ with module.codegen.helm_release.codegen,
│ on modules/codegen/main.tf line 7, in resource "helm_release" "codegen":
│ 7: resource "helm_release" "codegen" {
│
`

Reply:
ok